### PR TITLE
pal refresh fixes

### PR DESCRIPTION
**Pull request by howard-stearns**
When you drop a connection (deliberately "forget" this user) in the People app, refresh the list so that the user disappears from your list without having to close/open.

When you refresh, go to the same scroll position you were at, and do NOT get any more pages of results than you need to (as if you had opened from the beginning and scrolled to that position, but no farther).

Fixes https://highfidelity.fogbugz.com/f/cases/16893/forgetting-a-connection-in-pal-doesn-t-show-you-that-it-did-anything (and duplicate https://highfidelity.fogbugz.com/f/cases/15911/)

**Test Plan**: Please check all the PAL connection tests. (No need to check nearby tests.)

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/11511/HighFidelity-Beta-PR13675-48184c1.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/11511/HighFidelity-Beta-PR13675-48184c1.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/11511/HighFidelity-Beta-Interface-PR13675-48184c1.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11511/HighFidelity-Beta-PR13675-48184c1.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11511/HighFidelity-Beta-Interface-PR13675-48184c1.exe

File: scripts/system/pal.js [hunk 1]
```diff
@@ -361,8 +360,9 @@ function getProfilePicture(username, callback) { // callback(url) if successfull
         callback(matched[1]);
     });
 }
+var SAFETY_LIMIT = 600;
```
Comment (davidkelly) [inline scripts/system/pal.js]: we are upping it to 600?  Just checking - also curious if we can render that many, etc..

File: scripts/system/pal.js [hunk 2]
```diff
@@ -406,16 +408,14 @@ function getConnectionData(specificUsername, domain) { // Update all the usernam
...
         getAvailableConnections(domain, function (users) {
-            if (domain) {
-                users.forEach(function (user) {
-                    updateUser(frob(user));
-                });
-            } else {
-                sendToQml({ method: 'connections', params: users.map(frob) });
-            }
+            users.forEach(function (user) {
+                pdateUser(frob(user));
```
Comment (davidkelly) [inline scripts/system/pal.js]: looks like a typo - `pdateUser`?

Comment (howard-stearns) [reply scripts/system/pal.js]: yup! thanks

File: scripts/system/pal.js (hunk 1 above)
Comment (howard-stearns) [reply scripts/system/pal.js]: good point. I'll put it back to 400, which is what we were using before.  

BTW, This is not used to limit connections in general: it is used when getting the number of connections _in this domain at the moment_  (which is why I wanted to bump it up to higher than the current target for people in a domain), and for the number of connections that are an exact (sub) match for the specified username (which surely will never be that high... ;-) )

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/11593/HighFidelity-Beta-PR13675-699d164.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/11593/HighFidelity-Beta-Interface-PR13675-699d164.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11593/HighFidelity-Beta-PR13675-699d164.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11593/HighFidelity-Beta-Interface-PR13675-699d164.exe

**Comment by qlilaseguinolaza** [comment]
Currently testing

**Comment by qlilaseguinolaza** [comment]
PASS @conklin94122

**Pull request merged**